compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/antlr4/BallerinaLexer.java

```diff
@@ -1,4 +1,4 @@
-// Generated from BallerinaLexer.g4 by ANTLR 4.5.3
+// Generated from /home/kavindu/WSO2-GIT/test/ballerina-lang/compiler/ballerina-lang/src/main/resources/grammar/BallerinaLexer.g4 by ANTLR 4.5.3
```

Comment:
Shall we generate the parser from `compiler/ballerina-lang/src/main/resources/grammar/` directory

```bash
Navigate to grammar directory:
java -jar ~/Downloads/antlr-4.5.3-complete.jar *.g4 -package org.wso2.ballerinalang.compiler.parser.antlr4 -o ../../java/org/wso2/ballerinalang/compiler/parser/antlr4/
```

Reply:
This will get fixed, Once I combine All PRs. But it is a good practice to generate parser as @rdhananjaya mentioned.

Reply:
ACK.